sdk/attestation/azure-security-attestation/test/ut/policygetset_test.cpp

```diff
@@ -440,6 +443,11 @@ namespace Azure { namespace Security { namespace Attestation { namespace Test {
 
       testName += "_";
       testName += testInfo.param.TeeType.ToString();
+      if (suffixVotes == 2)
+      {
+        testName += "_LIVEONLY_";
+      };
+      //+"_LIVEONLY_";
```

Comment:
because in attestetion the UTs are overengineered , instead of having like everywhere else TEST(vv,ddd) someone decided that they can have one generic method and they can dinamically update the name of the test at runtime based on god knows what, and also not document whet they do there,  
The issue is that some combination of those things dosn't work properly with the new infra, and i caould not  determine where is the actual code for the test , forget trying to debug to fix it , like i've done for everyone else. I filed a bug and assigned it to Larry to fix. the fun thing is the code coverage is not affected byt those tests being live only.